### PR TITLE
Convert AlertManager alerts to Grafana Cloud alerts

### LINKED ISSUE #8109: Convert AlertManager alerts to Grafana alerts
### Problem

We'd like to move alerts from running in the cluster via AlertManager to Grafana Cloud.

### Solution

* Enumerate all of our custom alerts and ensure every metric is exported to Grafana Cloud
* Manually recreate those alerts in Grafana via clickops
* Ensure they are disabled by default until testing
* Export those alerts to a file and commit to source

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

This PR converts our custom alert manager alert rules to Grafana Cloud alert rules

- Add the alert rules yaml file exported from Grafana Cloud
- Adjust value related template expressions for proper rendering
- Fix incorrect metric names in custom grpc alert rules
- Fix RestHighCPU query expression to show correct percentage

**Related issue(s)**:

Fixes #8109

**Notes for reviewer**:

- Added `by (cluster, ...)` everywhere to the expression since we have the same namespace in different clusters, e.g., `mainnet` namespace in both `mainnet-na` and `mainnet-eu`
- There is button in Grafana Cloud to export alert rules, but no import
- The yaml file contains two existing importer alert rules, `Preprod - ImporterFileVerificationErrors` and `Production - ImporterFileVerificationErrors`, we should remove them and use the new `ImporterFileVerificationErrors` inside Importer group however I need to coordinate with devops since those two are configured to sent to devops atm

Screenshot 2024-05-01 at 12 09 34 PM

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: docs/alert-rules/grafana-cloud-alert-rules.yaml [hunk 1]
```diff
@@ -0,0 +1,1924 @@
...
+        dashboardUid: dfca62a1-9d6c-4f21-9933-b60e8cdac7d2
+        panelId: 5
+        noDataState: OK
+        execErrState: OK
+        for: 2m
+        annotations:
+          __dashboardUid__: dfca62a1-9d6c-4f21-9933-b60e8cdac7d2
+          __panelId__: "5"
+          description: "mirror-importer file verification error rate of {{ humanizePercentage $values.B.Value }} is above the 10% threshold. Stream file type: {{ $labels.type }} stream. Pod: {{ $labels.namespace }}/{{ $labels.pod }}"
+          summary: "{{ $labels.type }} file verification error rate exceeds 5%"
```
Comment (edwin-greene) [inline docs/alert-rules/grafana-cloud-alert-rules.yaml:65]: The description says the error rate is above the 10% threshold, but the summary says it exceeds 5%. Should they be the same?

Comment (xin-hedera) [reply docs/alert-rules/grafana-cloud-alert-rules.yaml:65]: good catch, this is one of the two pre-existing alert rules, I need to work with devops to remove and switch the notification to the new one

**Review by edwin-greene — APPROVED** [review]
Looks good to me

**Review by jnels124 — DISMISSED** [review]
LGTM

File: docs/alert-rules/grafana-cloud-alert-rules.yaml [hunk 2]
```diff
@@ -0,0 +1,1924 @@
...
+    rules:
+      - uid: edjv2xbh8d6v4b
+        title: GrpcErrors
+        condition: A
+        data:
+          - refId: A
+            relativeTimeRange:
+              from: 600
+              to: 0
+            datasourceUid: grafanacloud-prom
```
Comment (steven-sheehy) [inline docs/alert-rules/grafana-cloud-alert-rules.yaml:86]: Is there a way to templatize the datasource or can we leave it out and let it choose the default? This alert config should potentially be portable to other operators.

File: docs/alert-rules/grafana-cloud-alert-rules.yaml [hunk 3]
```diff
@@ -0,0 +1,1924 @@
...
+        labels:
+          application: hedera-mirror-importer
+          area: downloader
+          env-category: production
+          runbook: https://www.notion.so/swirldslabs/mirrornode-importer-ImporterFileVerificationErrors-17d18f568f054663bd7b2c53e05d620a?pvs=4
+          severity: critical
+        isPaused: false
+  - orgId: 1
+    name: Grpc
+    folder: MirrorNode
```
Comment (steven-sheehy) [inline docs/alert-rules/grafana-cloud-alert-rules.yaml:75]: We should match the folder structure of the dashboards which has this as `Mirror`.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: docs/alert-rules/grafana-cloud-alert-rules.yaml
[the remark this answers is not in the document]
Comment (xin-hedera) [reply docs/alert-rules/grafana-cloud-alert-rules.yaml:1]: moved the file

File: docs/alert-rules/README.md
[the remark this answers is not in the document]
Comment (xin-hedera) [reply docs/alert-rules/README.md:1]: removed

File: docs/alert-rules/grafana-cloud-alert-rules.yaml [hunk 4]
```diff
@@ -0,0 +1,1924 @@
...
+          area: resource
+          severity: critical
+        isPaused: false
+  - orgId: 1
+    name: Default 1m
+    folder: MirrorNode Preprod
+    interval: 1m
+    rules:
+      - uid: d7863a89-93c7-4830-a2df-67b6b9f081db
+        title: Preprod - ImporterFileVerificationErrors
```
[the remark this answers is not in the document]
Comment (xin-hedera) [reply docs/alert-rules/grafana-cloud-alert-rules.yaml]: these two existing rules are deleted

File: docs/alert-rules/grafana-cloud-alert-rules.yaml (hunk 2 above)
Comment (xin-hedera) [reply docs/alert-rules/grafana-cloud-alert-rules.yaml]: don't see how to template it. we can try to omit `datasourceUid` and it probably will set to the default data source, however will leave it as is and test it when later we try to figure out how to import

File: docs/alert-rules/grafana-cloud-alert-rules.yaml (hunk 3 above)
Comment (xin-hedera) [reply docs/alert-rules/grafana-cloud-alert-rules.yaml]: done

**Pull request merged**